aten/src/ATen/native/TensorShape.cpp

```diff
@@ -3527,7 +3526,7 @@ std::vector<Tensor> meshgrid(TensorList tensors,
   // * Why do we even support this function for exactly one input?
   bool swap_first_and_second_tensors = false;
 
-  if (indexing == "xy") {
+  if (*indexing == "xy") {
```

Comment:
nit: would be better to rename the optional `maybe_indexing` and have a non optional `indexing`

Reply:
done!